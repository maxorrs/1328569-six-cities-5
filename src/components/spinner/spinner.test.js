import React from 'react';
import renderer from 'react-test-renderer';
import Spinner from './spinner';

describe(`Spinner is rendered correctly`, () => {
  it(`Spinner is rendered correctly`, () => {
    const tree = renderer
      .create(
          <Spinner />
      )
    .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
