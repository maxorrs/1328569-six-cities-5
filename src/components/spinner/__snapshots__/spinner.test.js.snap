// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner is rendered correctly Spinner is rendered correctly 1`] = `
<div
  className="loadingio-spinner-rolling-iwj80ro4ny"
>
  <div
    className="ldio-mhj2lepa3va"
  >
    <div />
  </div>
</div>
`;
